src/commands/module-loadex.json

```
@@ -0,0 +1,53 @@
+{
+    "LOADEX": {
+        "summary": "Load a module with extended parameters",
+        "complexity": "O(1)",
+        "group": "server",
+        "since": "7.0.0",
+        "arity": -4,
```

```suggestion
        "arity": -3,
```
Only `MODULE LOADEX PATH` is required now.